ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.16-nightly-x86.yaml

```diff
@@ -52,35 +52,15 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-azure-ipi-ovn-ipsec
   timeout: 5h0m0s
-- always_run: false
```

Comment:
probably should keep this as a possible test if we ever need to go back and rerun the data path tests for comparison to 4.17 on azure

Reply:
@paigerube14 I removed it because we used 'azure-ipsec' profile instead of pure 'azure' now, that is the same in 4.17 config yaml. As you think we should keep pure 'azure', I will add it back here and also add it to 4.17 config yaml.